report_thesis/src/sections/background/linear_and_regularization_models/pls.tex

```diff
@@ -1,46 +1,23 @@
-\subsubsection{Partial Least Squares (PLS)}
-We now describe \gls{pls} based on \citet{James2023AnIS}.
+\subsubsection{Principal Component Regression (PCR) \& Partial Least Squares (PLS)}\label{subsec:pls}
 In order to understand \gls{pls}, it is helpful to first consider \gls{pcr}, as \gls{pls} is an extension of \gls{pcr} that aims to address some of its limitations.
+We provide provide an overview of both regression techniques based on \citet{James2023AnIS}.
 
 \gls{pcr} extends \gls{pca} in the context of regression analysis.
-In \gls{pcr}, the dataset $\mathbf{X}$ is decomposed using \gls{pca} as:
+First, \gls{pca} is performed to identify the $k$ principal components that capture the most variance in the data.
+These components are then used in a linear regression model to predict the target variable by fitting a linear model via least squares regression.
+The key intuition behind \gls{pcr} is that a small number of principal components are sufficient to capture most of the variance in the data, which can then be used to predict the target variable.
+That is, the directions of the principal components are assumed to be associated with the target variable.
```

Comment:
```suggestion
That is, the directions of the principal components are assumed to be associated with the target variables.
```

Reply:
Disagree. "Target variable" is correct if predicting a single outcome, as is typical in Principal Component Regression, while "target variables" would imply multiple outcomes.